packages/node-integration-tests/package.json

```diff
@@ -10,16 +10,20 @@
     "durability": "yarn ts-node src/durability-tests/durability-tests.ts",
     "concurrently": "yarn run -T concurrently",
     "ts-node": "yarn run -T ts-node",
-    "test": "yarn perf && yarn durability"
+    "test": "yarn perf && yarn durability",
+    "cloudflare-workers": "yarn run -T jest src/cloudflare"
```

Comment:
Nit: maybe "test:cf-workers" would be more consistent?

Reply:
Totally! Still make sense to keep cloudflare workers out of the main `test` script? Originally had them in, only reason to keep out is that they require a higher node.js runtime that the other tests do.

Reply:
Maybe we just remove the basic test script altogether?